ffc-ahwr-event/event/event.js

```diff
@@ -11,7 +12,7 @@ const saveEvent = async (context, event) => {
   const timespan = new Date(raisedEvent.action.raisedOn).getTime()
 
   const partitionKey = `${sbi}`
-  let rowKey = `${partitionKey}_${timespan}`
+  let rowKey = buildRowKey(partitionKey, timespan, eventType)
```

Comment:
Hate to say but it feels a bit unnecessary to abstract this into a function. A template string tells me then and there what the structure of the rowKey is.

Reply:
I abstracted it that way for the unit test

Reply:
but if you didn't abstract it away we wouldn't have to write and maintain tests for it 😄  The outcome is already tested in `event.test.js`.

Reply:
Agree with Matt, the separate function doesn't really give us anything. The 1 liner in this file, plus the unit test in test/unit/event/event.test.js should be sufficient